src/subcommands/lexical-search.go

```diff
@@ -0,0 +1,104 @@
+package subcommands
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"sort"
+	"strings"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+type LexicalSearchArgs struct {
+	IndexPath  string
+	SearchTerm string
+	TopN       int
+}
+
+type FileOccurrence struct {
```

Comment:
make types unexported by default if they are internal implementation details that don't need to be accessed outside the file/test/etc

Reply:
done